### PR TITLE
[JBPM-5287] properly Mavenize jbpm-installer

### PR DESCRIPTION
**Pull request by psiroky**
I basically took all the resources in the jbpm-installer dir and moved them under src/main/resources. Also new assembly zip is produced as part of the build (not sure how we were creating the zip before?)

Most of the files were also using CRLF (Windows) like endings. This got fixed as sideproduct of the move (git basically automatically fixed those line endings when moving the files).

https://github.com/droolsjbpm/jbpm/pull/582/files?w=0 shows the actual code/POM changes.

@krisv or @mswiderski could you please take a look?

### CODE REVIEW

File: jbpm-installer/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java [hunk 1]
```diff
@@ -80,7 +81,7 @@ public void testPersistedProcess(String type) throws IOException, ParseException
             scriptRunnerContext.persistOldProcessAndSession(TEST_SESSION_ID, TEST_PROCESS_ID, TEST_PROCESS_INSTANCE_ID);
             scriptRunnerContext.createSomeTask();
             // Execute upgrade scripts.
-            scriptRunnerContext.executeScripts(new File(getClass().getResource("/upgrade-scripts").getFile()), type);
+            scriptRunnerContext.executeScripts(new File(getClass().getResource("/db/upgrade-scripts").getFile()), type);
```
Comment (mswiderski) [inline jbpm-installer/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java]: why not use DB_UPGRADE_SCRIPTS_RESOURCE_PATH here as well?

**Comment by mswiderski** [comment]
@psiroky just a tiny comment and besides that +1 from me

File: jbpm-installer/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java (hunk 1 above)
Comment (psiroky) [inline jbpm-installer/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java]: Good point. No idea why I did like that :) will fix it

Comment (psiroky) [inline jbpm-installer/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java]: Fixed

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
+1

**Pull request merged**